### PR TITLE
Allow combined selectors

### LINKED ISSUE #198: Combined selector with :root
Hello !

I have an issue with a combined css selector

## Input CSS:
```
:root,
[data-theme=light] {
  --color-red: #A00;
}
body {
  color: var(--color-red);
}
```
## Expected output with `preserve: true`:
```
:root,
[data-theme=light] {
  --color-red: #A00;
}
body {
  color: #A00;
  color: var(--color-red);
}
```
## Expected output with `preserve: false`:
```
body {
  color: #A00;
}
```

### PR DESCRIPTION
**Pull request by ryuran**
Fix #198

### CODE REVIEW

File: src/lib/get-custom-properties-from-root.js [hunk 1]
```diff
@@ -48,8 +48,8 @@ const rootSelectorRegExp = /^:root$/i;
 const customPropertyRegExp = /^--[A-z][\w-]*$/;
 
 // whether the node is an html or :root rule
-const isHtmlRule = node => node.type === 'rule' && htmlSelectorRegExp.test(node.selector) && Object(node.nodes).length;
-const isRootRule = node => node.type === 'rule' && rootSelectorRegExp.test(node.selector) && Object(node.nodes).length;
+const isHtmlRule = node => node.type === 'rule' && -1 !== node.selector.split(',').findIndex(item => htmlSelectorRegExp.test(item)) && Object(node.nodes).length;
```
Comment (jonathantneal) [inline src/lib/get-custom-properties-from-root.js]: Thanks for finding this, @ryuran. I think this function can be simplified.

```suggestion
const isHtmlRule = node => node.type === 'rule' && -1 !== node.selectors.some(item => htmlSelectorRegExp.test(item)) && Object(node.nodes).length;

File: src/lib/get-custom-properties-from-root.js [hunk 2]
```diff
@@ -48,8 +48,8 @@ const rootSelectorRegExp = /^:root$/i;
...
+const isRootRule = node => node.type === 'rule' && -1 !== node.selector.split(',').findIndex(item => rootSelectorRegExp.test(item)) && Object(node.nodes).length;
```
Comment (jonathantneal) [inline src/lib/get-custom-properties-from-root.js]: I think this function can be simplified as well.

```suggestion
const isRootRule = node => node.type === 'rule' && -1 !== node.selectors.some(item => rootSelectorRegExp.test(item)) && Object(node.nodes).length;

**Comment by ryuran** [comment]
@jonathantneal I change `.findIndex()` to `.some()` but I think the `.split(',')` stay necessary.
`.some()` return a boolean, so I remove `-1 !==`

**Comment by ryuran** [comment]
👆 rebase

**Comment by ai** [comment]
@Semigradsky can we also release this fix?

**Pull request merged**